onnxruntime/python/tools/tensorrt/perf/benchmark.py

```diff
@@ -1608,15 +1577,14 @@ def main():
     args = parse_arguments()
     setup_logger(False)
     pp = pprint.PrettyPrinter(indent=4)
-
+    
     logger.info("\n\nStart perf run ...\n")
 
     models = {}
     parse_models_helper(args, models)
 
     if not os.path.exists("symbolic_shape_infer.py"):
-        p1 = subprocess.Popen(["sudo", "wget", "https://raw.githubusercontent.com/microsoft/onnxruntime/master/onnxruntime/python/tools/symbolic_shape_infer.py"])
-        p1.wait()
+        subprocess.run(["wget", "https://raw.githubusercontent.com/microsoft/onnxruntime/master/onnxruntime/python/tools/symbolic_shape_infer.py"], check=True)
```

Comment:
do we need to download it? this script is already under python/tools , can't we just use relative path to call the symbolic shape infer script?

Reply:
Yes, good catch.